doc/book/box/data_model.rst

```diff
@@ -554,6 +554,104 @@ Charts explaining the precise differences from DUCET order are
 in the
 `Common Language Data Repository <https://unicode.org/cldr/charts/30/collation>`_.
 
+.. _index-box_foreign_keys:
+
+------------
+Foreign keys
+------------
+
+**Foreign keys** provide links between related spaces, therefore maintaining the
+`referential integrity <https://en.wikipedia.org/wiki/Referential_integrity>`_
+of the database.
+
+Some fields can only contain values present in other spaces. For example,
+shop orders always belong to existing customers. Hence, all values of the ``customer``
+field of the ``orders`` space must exist in the ``customers`` space. In this case,
+``customers`` is a **parent space** for ``orders`` (its **child space**). Foreign keys
+enable automatic checks of values presence in the parent space upon each insertion and
+modification of the child space.
```

Comment:
I think this is a very complicated sentence because it's hard for me to understand. Consider rephrasing it with "when/each time" and fewer verbal nouns.